components/shared/SearchInput.tsx

```diff
@@ -0,0 +1,86 @@
+import { RiSearchLine } from 'react-icons/ri';
+import dynamic from 'next/dynamic';
+import {  GroupBase, StylesConfig } from 'react-select';
+
+const AsyncReactSelect = dynamic(() => import('react-select/async'), {
+  ssr: false,
+});
+
+const defaultStyles: StylesConfig<unknown, boolean, GroupBase<unknown>> | undefined = {
+  dropdownIndicator: () => ({
+    display: 'none',
+  }),
+  indicatorSeparator: () => ({
+    display: 'none',
+  }),
+  placeholder: (provided) => ({
+    ...provided,
+    color: 'rgba(193, 193, 193, 1)',
```

Comment:
Is it possible to move these styles out of txs and to .less file?
Or maybe use classes and style them in .less:
https://react-select.com/styles#the-classnames-prop